### PR TITLE
[Doc] MV BACKUP Doc

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/en/administration/Backup_and_restore.md [hunk 1]
```diff
@@ -171,6 +171,34 @@ You can optimize the performance of BACKUP or RESTORE jobs by modifying the foll
 | download_worker_count   | The maximum number of threads for the download tasks of RESTORE jobs on a BE node. Default: `1`. Increase the value of this configuration item to increase the concurrency of the download task. |
 | max_download_speed_kbps | The upper limit of the download speed on a BE node. Default: `50000`. Unit: KB/s. Usually, the speed of the download tasks in RESTORE jobs will not exceed the default value. If this configuration is limiting the performance of RESTORE jobs, you can increase it according to your bandwidth.|
 
+## Materialized view BACKUP and RESTORE
+
+During a BACKUP or a RESTORE job of a table, StarRocks automatically backs up or restores its [Synchronous materialized view](../using_starrocks/Materialized_view-single_table.md).
+
+From v3.2.0, StarRocks supports backing up and restoring [asynchronous materialized views](../using_starrocks/Materialized_view.md) when you backing up and restoring the database they reside in.
```
Comment (evelynzhaojie) [inline docs/en/administration/Backup_and_restore.md:178]: grammar: when you back up and restore

File: docs/zh/administration/Backup_and_restore.md [hunk 2]
```diff
@@ -203,6 +203,34 @@ PROPERTIES (
...
+
+在备份或还原表（Table）数据期间，StarRocks 会自动备份或还原其中的 [同步物化视图](../using_starrocks/Materialized_view-single_table.md)。
+
+从 v3.2.0 开始，StarRocks 支持在备份和还原数据库（Database）时备份和还原数据库中的 [异步物化视图](../using_starrocks/Materialized_view.md)。
+
+在备份和还原数据库期间，StarRocks 执行以下操作：
+
+- **BACKUP**
+
+1. 遍历数据库以收集所有表和异步物化视图的信息。
```
Comment (evelynzhaojie) [inline docs/zh/administration/Backup_and_restore.md:216]: 这里不包含同步物化视图吗

File: docs/en/administration/Backup_and_restore.md [hunk 3]
```diff
@@ -171,6 +171,34 @@ You can optimize the performance of BACKUP or RESTORE jobs by modifying the foll
...
+
+During BACKUP and RESTORE a database, StarRocks does as follows:
+
+- **BACKUP**
+
+1. Traverse the database to gather information on all tables and asynchronous materialized views.
+2. Adjust the order of tables in the BACKUP and RESTORE queue, ensuring that the base tables of materialized views are positioned before the materialized views:
+   - If the base table exists in the current database, StarRocks adds the table to the queue.
+   - If the base table does not exist in the current database, StarRocks prints a warning log and proceed with the BACKUP operation without blocking the process.
```
Comment (evelynzhaojie) [inline docs/en/administration/Backup_and_restore.md:187]: proceeds

File: docs/zh/administration/Backup_and_restore.md (hunk 2 above)
Comment (EsoragotoSpirit) [reply docs/zh/administration/Backup_and_restore.md:216]: 同步物化视图跟随各自的基表备份还原

File: docs/en/administration/Backup_and_restore.md [hunk 4]
```diff
@@ -171,6 +171,34 @@ You can optimize the performance of BACKUP or RESTORE jobs by modifying the foll
...
+3. Execute the BACKUP task in the order of the queue.
+
+- **RESTORE**
+
+1. Restore the tables and materialized views in the order of the BACKUP and RESTORE queue.
+2. Re-build the dependency between materialized views and their base tables, and re-submit the refresh task schedule.
+
+Any error encountered throughout the RESTORE process will not block the process.
+
+After RESTORE, you can check the state of the materialized view using [SHOW MATERIALIZED VIEWS](../sql-reference/sql-statements/data-manipulation/SHOW_MATERIALIZED_VIEW.md).
```
Comment (evelynzhaojie) [inline docs/en/administration/Backup_and_restore.md:197]: wording: check the status

File: docs/en/administration/Backup_and_restore.md [hunk 5]
```diff
@@ -171,6 +171,34 @@ You can optimize the performance of BACKUP or RESTORE jobs by modifying the foll
...
+
+- If the materialized view is active, it can be used directly.
+- If the materialized view is inactive, it might be because its base tables are not restored. After all the base tables are restored, you can use [ALTER MATERIALIZED VIEW](../sql-reference/sql-statements/data-definition/ALTER_MATERIALIZED_VIEW.md) to re-activate the materialized view.
```
Comment (evelynzhaojie) [inline docs/en/administration/Backup_and_restore.md:200]: After all the base tables of the materialized view are restored

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Pull request merged**